docs/how-to-create-a-machine.md

```diff
@@ -6,20 +6,22 @@ We will represent a machine as such
 
 ```
 a ┌────┐ b
-──┤    ├──
+──┤    ├─>
   └────┘
 ```
 
 as black boxes, where inputs are on the left and outputs are on the right.
 
-The `StateMachine a b` data type has four constructors which we can use to construct a machine:
+The `StateMachine a b` data type has six constructors which we can use to construct a machine:
 
 - `Basic`
 - `Compose`
 - `Parallel`
 - `Alternative`
+- `Feedback`
+- `Kleisli`
 
-Let's start with the last three.
+Let's start with the last five.
```

Comment:
```suggestion
Every state machine that is not `Basic` is a composition of two state machines, which are ultimately expressed in terms of `Basic`.

Let's skip over `Basic` for now, and look at the other constructors first.
```

This is why I think `Compose` should be named `Sequence`; all of these constructors are compositions, but not every composition is sequential.

Reply:
yes, now I agree with you. I'll do the change in a separate PR